CODEOWNERS

```diff
@@ -117,7 +120,8 @@ drivers/timer/riscv_machine_timer.c      @nategraff-sifive @kgugala @pgielda
 drivers/usb/                             @jfischer-phytec-iot @finikorg
 drivers/usb/device/usb_dc_stm32.c        @ydamigos @loicpoulain
 drivers/i2c/i2c_ll_stm32*                @ldts @ydamigos
-drivers/wifi/eswifi                      @loicpoulain
+drivers/wifi/                            @jukkar @tbursztyka @pfalcon
+drivers/wifi/eswifi                      @jukkar @tbursztyka @pfalcon @loicpoulain
```

Comment:
when it goes down to individual drivers I propose to just have the driver maintainer, otherwise it does not scale. 
You can add yourself to the driver level if you want, but I would not add other people.

Reply:
> You can add yourself to the driver level if you want, but I would not add other people.

Well, the idea here is to improve PR visibility and review. Relying on "myself" doesn't help. Suppose driver maintainer submits patch, what happens next? Nothing, for a while.

There're other ways to resolve that situation, e.g. to define a networking review team and duplicate just its name instead of quite a long string of codereviewers. But as you doesn't mention it, as @jukkar and @tbursztyka doesn't respond at all, let me just remove that line.